### PR TITLE
Correctly install go1.14 in docker image

### PR DESCRIPTION
**Pull request by chaodaiG**
/assign chizhg
/cc vagababov

### CODE REVIEW

File: images/prow-tests-go114/Dockerfile [hunk 1]
```diff
@@ -15,26 +15,11 @@
...
-ENV GOARCH "amd64"
-
-# install netbase as it's required for Go linux amd64 install (OS specific)
-RUN apt-get install netbase
-
-# go1.14 is not released yet. Downloading the beta version directly from github and building from source
-# Reference: https://golang.org/doc/install/source
-RUN mv /usr/local/go /usr/local/go-old && \
-    wget -q "https://github.com/golang/go/archive/${GO_TARBALL}" && \
+# Install go at 1.12 same as how it's installed in kubekins-e2e image, reference code here:
```
Comment (chizhg) [inline images/prow-tests-go114/Dockerfile:18]: 1.12 -> 1.14

Comment (chaodaiG) [reply images/prow-tests-go114/Dockerfile]: fixed

**Comment by chizhg** [comment]
@googlebot rescan

**Pull request merged**